guides/doc-Release_Notes/topics/katello-4.9.2.adoc

```diff
@@ -0,0 +1,57 @@
+= Katello 4.9.2
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1752[Redmine]
+
+== Katello
+
+=== Alternate Content Sources
+
+* Custom ACS path help text is missing file:// - https://projects.theforeman.org/issues/36631[#36631]
+
+=== Client/Agent
+
+* tasks Actions::Katello::BulkAgentAction without any sub-plans and stuck in running/pending - https://projects.theforeman.org/issues/36528[#36528]
+* Add UI banners warning users about katello-agent removal - https://projects.theforeman.org/issues/36526[#36526]
+
+=== Content Views
+
+* undefined method \'#single_content_view\' for Katello::Host::ContentFacet::Jail (Katello::Host::ContentFacet) (Safemode::NoMethodError) - https://projects.theforeman.org/issues/36684[#36684]
```

Comment:
The script generates incorrect escaping. From https://theforeman-foreman-documentation-preview-pr-2479.surge.sh/3.7/Release_Notes/index-katello.html#_content_views
```
undefined method \'#single_content_view\' for Katello::Host::ContentFacet::Jail (Katello::Host::ContentFacet) (Safemode::NoMethodError) - [#36684](https://projects.theforeman.org/issues/36684)
```

In the past I manually dealt with this. https://github.com/theforeman/foreman-documentation/pull/2427 should properly solve it.

Could you go through the rendered files and check this?